scripts/jobAdd.js

```diff
@@ -109,3 +110,4 @@ Projects.getData = function(callback) {
 
 Employment.extractAll();
 Projects.getData(Projects.loadEm);
+})(window);
```

Comment:
you need to pass your objects into the global namespace here
remember:
`module.Employment = Employment`